P8/DataConverters/ShapeletHistogramConverter/FeatureExtraction/src/FeatureFinding.h

```diff
@@ -54,11 +56,23 @@ namespace FeatureFinding {
         if (windows.empty())
             throw std::logic_error("Missing windows.");
 
+        const std::vector<Attribute*> attributes {
+            new Frequency(0.01),
+            new Frequency(0.02),
+            new Frequency(0.04),
+            new Frequency(0.08),
+            new Frequency(0.1),
+            new Frequency(0.2),
+            new Frequency(0.4),
+            new Frequency(0.8),
+            new MinDist()
+        }; // Intentionally not freed
```

Comment:
I suppose this should be some kind of option somewhere.
But its fine for now, it can be done later.